concrete/src/File/FileListTransformer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Concrete\Core\File;
+
+use League\Fractal\TransformerAbstract;
+
+class FileListTransformer extends TransformerAbstract
+{
+    public function transform(FileList $fileList)
+    {
+        $results = [];
+
+        foreach ($fileList->getPagination()->getCurrentPageResults() as $file) {
+            $results[] = $file->getRecentVersion()->getJSONObject();
```

Comment:
It might be nice to break this out into a protected method so that people can extend change how the json is outputed by overriding a simple method like:
```
protected function transformFile(File $file)
{
    return $file->getRecentVersion()->getJsonObject();
}
```

Instead of needing to replace the entire transform method

Reply:
Pulling this out.